src/components/Blog/Blog.tsx

```diff
@@ -4,23 +4,39 @@ import { Header, Profile, SideNavigate } from '../Layouts';
 import { UserQueries } from '@/apis/user';
 import { BlogQueries } from '@/apis/blog';
 import PrivateBlogList from './BlogList/PrivateBlogList';
+import useScrollPosition from '@/hook/useScrollPosition';
+import PublicBlogList from './BlogList/PublicBlogList';
 
-export default function Blog() {
+export default function Blog({ type }: { type: 'public' | 'private' }) {
```

Comment:
```suggestion
interface BlogProps {
  type: 'public' | 'private';
}

export default function Blog({ type }: BlogProps) {
```
개인적으로 저는 코드가 좀 더 길어져도 타입을 직접 명시하기보다는 인터페이스(혹은 type)를 사용하는 것을 더 선호합니다.

Reply:
@bluetree7878 리뷰 감사합니다. 저도 이 부분은 동의합니다. 타입명 지정하기 귀찮아서 나중에 해야지 하면서 안하게 되는 경우가 많네요😓 담 부턴 타입 지정 후 진행해야 겠어요